js/apps/kc-create/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I am not seeing a [`bin`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin) field here, which is needed to expose a binary that can be used by consumers.